src/plugins/dataflow/nodes/controls/dropdown-list-control.tsx

```diff
@@ -172,7 +175,13 @@ export class DropdownListControl extends Rete.Control {
   };
 
```

Comment:
I'm not sure why - but the original, functionality identical but more terse version of this was raising "TypeErrors" during runtime (So - not a type error in TypeScript, but something in the execution called a TypeError?).  Writing it with more procedural checks eliminated that.